tests/sqlparser_common.rs

```diff
@@ -2103,18 +2108,114 @@ fn parse_select_order_by_nulls_order() {
                 expr: Expr::Identifier(Ident::new("lname")),
                 asc: Some(true),
                 nulls_first: Some(true),
+                with_fill: None,
             },
             OrderByExpr {
                 expr: Expr::Identifier(Ident::new("fname")),
                 asc: Some(false),
                 nulls_first: Some(false),
+                with_fill: None,
             },
         ],
         select.order_by
     );
     assert_eq!(Some(Expr::Value(number("2"))), select.limit);
 }
 
+#[test]
```

Comment:
given this is clickhouse specific should these tests rather live in `sqlparser_clickhouse.rs`?